src/jvm/test/org/jetbrains/kotlinx/lincheck_test/gpmc/ModelCheckerTest.kt

```diff
@@ -101,6 +101,6 @@ private class CollectResultsVerifier : Verifier {
     }
 }
 
-private const val DEFAULT_INVOCATIONS_COUNT = 100
+private const val DEFAULT_INVOCATIONS_COUNT = 500
 
-internal const val TIMEOUT = 30_000L // 30 sec
\ No newline at end of file
+internal const val TIMEOUT = 60_000L // 60 sec
```

Comment:
Why do you need to increase the TL?

Reply:
After increasing the default invocation count because of the [mentioned reason](https://github.com/JetBrains/lincheck/pull/460/files/84f3aab419fe89a50a76922649882a671d32d7dd#r1936469190), tests may last longer than before.

Reply:
I do not see any significant difference either on TeamCity or locally.